docs/api.md

```diff
@@ -327,7 +328,35 @@ will be closed.
 If no URLs are specified, this method returns all cookies.
 If URLs are specified, only cookies that affect those URLs are returned.
 
-> **NOTE** the default browser context cannot be closed.
+#### browserContext.evaluateOnNewDocument(pageFunction[, ...args])
+- `pageFunction` <[function]|[string]> Function to be evaluated in all pages in the browser context
+- `...args` <...[Serializable]> Arguments to pass to `pageFunction`
+- returns: <[Promise]>
+
+Adds a function which would be invoked in one of the following scenarios:
+- whenever the page is navigated
+- whenever the child frame is attached or navigated. In this case, the function is invoked in the context of the newly attached frame
+
+The function is invoked after the document was created but before any of its scripts were run. This is useful to amend  the JavaScript environment, e.g. to seed `Math.random`.
+
+An example of overriding the navigator.languages property before the page loads:
+
+```js
+// preload.js
+
+// overwrite the `languages` property to use a custom getter
+Object.defineProperty(navigator, "languages", {
```

Comment:
We have a better support for overriding locale, so maybe rewrite the example to use aforementioned Math.random ?